src/XMakeBuildEngine/Logging/BaseConsoleLogger.cs

```diff
@@ -370,7 +370,7 @@ internal bool IsVerbosityAtLeast(LoggerVerbosity checkVerbosity)
         /// <summary>
         /// Sets foreground color to color specified
         /// </summary>
-        internal static void SetColor(ConsoleColor c)
+        public static void SetColor(ConsoleColor c)
```

Comment:
Is there a reason to make this public? Would like to keep internal if at all possible, doesn't seem like it needs to change.